docs/topics/forms/media.txt

```diff
@@ -2,7 +2,7 @@
 Form Assets (the ``Media`` class)
 =================================
 
-Rendering an attractive and easy-to-use Web form requires more than just
+Rendering an attractive and easy-to-use Web form requires more than
```

Comment:
I think this one can be reverted, it has the sense of "only" and "only" doesn't sound as good